.github/ISSUE_TEMPLATE/new-release.md

```diff
@@ -0,0 +1,132 @@
+---
```

Comment:
this will be removed before merging?

Reply:
Thanks, this is not part of our change and i think it came from some old merge i have removed it as its not there in develop, Thanks